### ISSUE TITLE
Rule 'array-bracket-even-spacing' does not seem to be working anymore

### ISSUE THREAD

**Issue by josemigallas** [standard/eslint-plugin-standard#31]
Since I upgrade my packages, Eslint is complaining about my brackets.:Here are the (most relevant?) packages I'm using:
```
"babel-eslint": "^10.0.1",
"eslint": "^5.15.1",
"eslint-config-standard": "^12.0.0",
"eslint-loader": "^2.1.1",
"eslint-plugin-standard": "^4.0.0",
```
And this one the error I get everywhere:
```
error  A space is required after '{'   object-curly-spacing
```

**Comment by feross** [comment]
It seems like the rule is working as intended?

Also, we have moved away from this rule in `standard` 13, so if you upgrade to this version then your errors should go away because we don't even use this rule anymore
